aaelf32/aaelf32.rst

```diff
@@ -0,0 +1,3289 @@
+   | 81   | :code:`R_ARM_LDC_SB_G0`    | Yes      | :code:`LDC`, :code:`STC`     | ABS(X) & G\ :sub:`0` (LDC)  |
+   +------+----------------------------+----------+------------------------------+-----------------------------+
+   | 82   | :code:`R_ARM_LDC_SB_G1`    | Yes      | :code:`LDC`, :code:`STC`     | ABS(X) & G\ :sub:`1` (LDC)  |
+   +------+----------------------------+----------+------------------------------+-----------------------------+
+   | 83   | :code:`R_ARM_LDC_SB_G2`    | Yes      | :code:`LDC`, :code:`STC`     | ABS(X) & G\ :sub:`2` (LDC)  |
+   +------+----------------------------+----------+------------------------------+-----------------------------+
+   | 84   | :code:`R_ARM_MOVW_BREL_NC` | No       | :code:`MOVW`                 | X & 0xFFFF                  |
+   +------+----------------------------+----------+------------------------------+-----------------------------+
+   | 85   | :code:`R_ARM_MOVT_BREL`    | No       | :code:`MOVT`                 | X & 0xFFFF0000              |
+   +------+----------------------------+----------+------------------------------+-----------------------------+
+   | 86   | :code:`R_ARM_MOVW_BREL`    | Yes      | :code:`MOVW`                 | X & 0xFFFF                  |
+   +------+----------------------------+----------+------------------------------+-----------------------------+
+   | 97   | :code:`R_ARM_GOT_BREL12`   | Yes      | :code:`LDR`                  | ABS(X) & 0xFFF              |
+   +------+----------------------------+----------+------------------------------+-----------------------------+
+   | 98   | :code:`R_ARM_GOTOFF12`     | Yes      | :code:`LDR`, :code:`STR`     | ABS(X) & 0xFFF              |
+   +------+----------------------------+----------+------------------------------+-----------------------------+
+   | 109  | :code:`R_ARM_TLS_LDO12`    | Yes      | :code:`LDR`, :code:`STR`     | ABS(X) & 0xFFF              |
+   +------+----------------------------+----------+------------------------------+-----------------------------+
+   | 110  | :code:`R_ARM_TLS_LE12`     | Yes      | :code:`LDR`, :code:`STR`     | ABS(X) & 0xFFF              |
+   +------+----------------------------+----------+------------------------------+-----------------------------+
+   | 111  | :code:`R_ARM_TLS_IE12GP`   | Yes      | :code:`LDR`                  | ABS(X) & 0xFFF              |
+   +------+----------------------------+----------+------------------------------+-----------------------------+
+
+The formation of the initial addend in a REL type relocation for the various
+instruction classes is described in the following table. Insn modification
+describes how the 32-bit result X is written back to the instruction;
+Result_Mask is the value of X after the masking operation described in `Static
+Arm instruction relocations`_ has been applied.
+
+.. _Arm relocation actions by instruction type:
+
+.. table:: Arm relocation actions by instruction type
+  :widths: 13 48 40
```

Comment:
as described above on the version history table, please use a class to specify the width